zinnia/views/archives.py

```diff
@@ -21,6 +21,8 @@
 from zinnia.views.mixins.tz_fixes import EntryDayTZFix
 from zinnia.views.mixins.tz_fixes import EntryWeekTZFix
 from zinnia.views.mixins.tz_fixes import EntryMonthTZFix
+from quoin.feature import models
```

Comment:
I'm sorry if I mislead you, but this isn't the right way to do this.  A third party app should not depend on our internal marketing site package.